examples/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.fits
+*.fits.fz
```

Comment:
Minor point, but I'd rather leave the examples/output/.gitignore alone and remove these.  We might want fits files in the repo at some point as inputs.  Ignoring everything in the output directory seems safe though.